tracer/src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -26,6 +26,7 @@
     the */** trick here ensures that there's at least one subdirectory, which indicates the files comes
     from a source generator as opposed to something that is coming from some other tool. -->
     <Compile Remove="$(GeneratedFolder)/*/**/*.cs" />
+    <Compile Update="Configuration\ConfigurationKeys.*.cs" DependentUpon="ConfigurationKeys.cs" />
```

Comment:
I've never seen this before, what does it do?

Reply:
It doesn't change the compilation at all, it just nests the files nicely in VS/Rider. Totally optional, thought it might be a nice to have, no problems removing it if it annoys anyone! 🙂 

![image](https://user-images.githubusercontent.com/18755388/206410455-61f2a279-af21-4856-825d-6b797680023b.png)